diff --git a/src/schema.rs b/src/schema.rs
index 8fc3be4..588aa7d 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -1,28 +1,28 @@
 // @generated automatically by Diesel CLI.
 
 diesel::table! {
     module_menu (tucan_id) {
-        tucan_id -> Array<Nullable<Int4>>,
+        tucan_id -> Array<Int4>,
         tucan_last_checked -> Timestamptz,
         name -> Text,
         normalized_name -> Text,
-        parent -> Nullable<Array<Nullable<Int4>>>,
+        parent -> Nullable<Array<Int4>>,
     }
 }
 
 diesel::table! {
     module_menu_module (module_menu_id, module_id) {
-        module_menu_id -> Array<Nullable<Int4>>,
-        module_id -> Array<Nullable<Int4>>,
+        module_menu_id -> Array<Int4>,
+        module_id -> Array<Int4>,
     }
 }
 
 diesel::table! {
     modules (tucan_id) {
-        tucan_id -> Array<Nullable<Int4>>,
+        tucan_id -> Array<Int4>,
         tucan_last_checked -> Timestamptz,
         title -> Text,
         module_id -> Text,
         credits -> Nullable<Int4>,
         content -> Text,
     }
