diff --git a/src/schema.rs b/src/schema.rs
index a1d75c9..c9d123a 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -1,26 +1,36 @@
 // @generated automatically by Diesel CLI.
 
 diesel::table! {
     module_menu_module (module_menu_id, module_id) {
-        module_menu_id -> Array<Nullable<Int8>>,
+        module_menu_id -> Array<Int8>,
         module_id -> Int8,
     }
 }
 
 diesel::table! {
     module_menu_unfinished (tucan_id) {
-        tucan_id -> Array<Nullable<Int8>>,
+        tucan_id -> Array<Int8>,
         tucan_last_checked -> Timestamptz,
         name -> Text,
         normalized_name -> Text,
-        parent -> Nullable<Array<Nullable<Int8>>>,
+        parent -> Nullable<Array<Int8>>,
         recursively_fetched -> Bool,
     }
 }
 
+diesel::table! {
+    module_menu (tucan_id) {
+        tucan_id -> Array<Int8>,
+        tucan_last_checked -> Timestamptz,
+        name -> Text,
+        normalized_name -> Text,
+        parent -> Nullable<Array<Int8>>,
+    }
+}
+
 diesel::table! {
     modules (tucan_id) {
         tucan_id -> Int8,
         tucan_last_checked -> Timestamptz,
         title -> Text,
         module_id -> Text,
